### PR TITLE
feat(new-primary-school): Implement conclusion page

### PR DESCRIPTION
**Pull request by birkirkristmunds**
[TS-815](https://dit-iceland.atlassian.net/browse/TS-815)

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts [hunk 1]
```diff
@@ -505,9 +505,11 @@ export const NewPrimarySchoolForm: Form = buildForm({
       ],
     }),
     buildFormConclusionSection({
-      alertTitle: newPrimarySchoolMessages.conclusion.alertTitle,
-      expandableHeader: newPrimarySchoolMessages.conclusion.nextStepsLabel,
-      expandableDescription: newPrimarySchoolMessages.conclusion.accordionText,
+      alertMessage: newPrimarySchoolMessages.conclusion.alertMessage,
```
Comment (veronikasif) [inline libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts:508]: I think you can remove this line, this is the same text that is default in the `buildFormConclusionSection`. Remember to also remove from `messages.ts`
```suggestion
```

File: libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts [hunk 2]
```diff
@@ -505,9 +505,11 @@ export const NewPrimarySchoolForm: Form = buildForm({
...
+      expandableHeader: newPrimarySchoolMessages.conclusion.expandableHeader,
```
Comment (veronikasif) [inline libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts:509]: I think you can remove this line, this is the same text that is default in the `buildFormConclusionSection`. Remember to also remove from `messages.ts`
```suggestion
```

File: libs/application/templates/new-primary-school/src/lib/messages.ts [hunk 3]
```diff
@@ -415,31 +415,24 @@ export const newPrimarySchoolMessages: MessageDir = {
...
     },
+    expandableDescription: {
+      id: 'dess.nps.application:conclusion.expandable.description#markdown',
+      defaultMessage:
+        'Skólastjóri mun taka afstöðu til umsóknarinnar eftir aðstæðum í skólanum hversu sinni og svara þér eins fljótt og auðið er í tölvupósti. Svarið mun einnig birtast í stafrænu pósthólfi hér á Island.is',
+      description:
+        'The principal will evaluate the application based on the current circumstances at the school and will respond to you as soon as possible by email. The response will also appear in the digital mailbox here on Island.is.',
+    },
+
     buttonsViewApplication: {
```
Comment (veronikasif) [inline libs/application/templates/new-primary-school/src/lib/messages.ts:436]: This text is not used, can you remove it? 😄

**Review by veronikasif — APPROVED** [review]
Small text changes, looks good otherwise! 💯

File: libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts (hunk 1 above)
Comment (birkirkristmunds) [reply libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts]: Removed

File: libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts (hunk 2 above)
Comment (birkirkristmunds) [reply libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm.ts]: Removed

File: libs/application/templates/new-primary-school/src/lib/messages.ts (hunk 3 above)
Comment (birkirkristmunds) [reply libs/application/templates/new-primary-school/src/lib/messages.ts]: Removed

**Pull request merged**